### PR TITLE
airtable doc, images, and navigation

### PR DESCRIPTION
**Pull request by Adorism**
## Change description

Airtable documentation created.
Images and Navigation added for this doc. 

Description here

### Development

- [X] Application changes have been tested appropriately

### Impact

- [ ] Code follows company security practices and guidelines
- [ ] Security impact of change has been considered
- [ ] Performance impact of change has been considered
- [ ] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [ ] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: pages/docs/integrations/grouparoo-airtable.mdx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+Companies use Airtable to organize their information in a user-friendly and collaborative environment that gives the simple structure of spreadsheets the additional functionality of a database.
+
+To [create a Destination](https://www.grouparoo.com/docs/config/destinations), you make choices about what data to sync.
+
+### Sync Modes
+
+A Destination’s sync mode determines what to update in a Destination. The following are supported by Mailjet:
+
+- **Sync**: Add, update, and remove Airtable Records as needed.This option looks for new records, changes to existing records, and deletions.
```
Comment (krishnaglick) [inline pages/docs/integrations/grouparoo-airtable.mdx]:
```suggestion
- **Sync**: Add, update, and remove Airtable Records. This option looks for new records, changes to existing records, and deletions.
```

File: pages/docs/integrations/grouparoo-airtable.mdx [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+- **Additive**: Add and update Airtable Records as needed, but do not remove anybody. This option looks for new Records to add to Airtable and changes to existing Airtable Records, but does not keep track of deletions.
```
Comment (krishnaglick) [inline pages/docs/integrations/grouparoo-airtable.mdx]: I might be suggestion things that were intentfully phrased.
```suggestion
- **Additive**: _Only_ add and update Airtable Records, no removal. This option looks for new Records to add to Airtable and changes to existing Airtable Records, but does not keep track of deletions.
```

File: pages/docs/integrations/grouparoo-airtable.mdx [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+- **Enrich**: Only update those Records that already exist in Airtable. Do not add or remove Records. This option will only update existing Records in Airtable.
+
+### Destination Options
+
+To further clarify how Intercom should sync data, there are some options you should choose that designate details about Airtable’s organization.
```
Comment (krishnaglick) [inline pages/docs/integrations/grouparoo-airtable.mdx]: `Intercom`?

Comment (Adorism) [reply pages/docs/integrations/grouparoo-airtable.mdx]: oops, I'll definitely change that.

File: pages/docs/integrations/grouparoo-airtable.mdx [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+        navigating to the{" "}
+        <a href="https://airtable.com/account">account page</a>.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Airtable Base</strong> (required)
+      </td>
+      <td>
+        This designates the Airtable Base to sync data to. Each airtable base
```
Comment (teallarson) [inline pages/docs/integrations/grouparoo-airtable.mdx:78]:
```suggestion
        This designates the Airtable Base to sync data to. Each Airtable Base
```

**Review by teallarson — APPROVED** [review]
Looks good!  Finding out those URL bits for the options was tricky.  A screenshot of a URL with the prefixes highlighted/circled (and the actual values redacted so people don't sync to ours) to show how to parse the URL would help users a bit more.  I don't think it's blocking.

**Review by krishnaglick — APPROVED** [review]

**Pull request merged**